### PR TITLE
adds cards info

### PR DESCRIPTION
**Pull request by rafibomb**

### CODE REVIEW

**Comment by rafibomb** [comment]
@canterberrie Can you look this over for merge?

File: src/building-blocks/card-info/card-info.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    border-radius: $card-info-border-radius;
+  }
+
+  .card-info-label-text {
+    color: $white;
+    font-size: 0.75rem;
+    font-weight: bold;
+    position: relative;
+    right: -40px;
+    top: 2px;
```
Comment (IamManchanda) [inline src/building-blocks/card-info/card-info.scss]: rem please ??

File: src/building-blocks/card-info/card-info.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<div class="card-info success radius">
```
Comment (canterberrie) [inline src/building-blocks/card-info/card-info.html]: is card-info a specific enough class?

File: src/building-blocks/card-info/card-info.scss [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    width: 0px;
+    transform:rotate(360deg);
+  }
+
+  .card-info-content {
+    padding: 0.5rem 1.5rem 0.875rem;
+  }
+
+  &.radius {
+    border-radius: $card-info-border-radius;
```
Comment (canterberrie) [inline src/building-blocks/card-info/card-info.scss]: What's the thinking behind using "radius" as a class? why not scope this into the container?

File: src/building-blocks/card-info/card-info.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: 'Info Card With Lables'
```
Comment (canterberrie) [inline src/building-blocks/card-info/card-info.yml]: Typo! "Labels" :P

File: src/building-blocks/card-info/card-info.scss (hunk 3 above)
Comment (rafibomb) [inline src/building-blocks/card-info/card-info.scss]: this is how callouts and buttons handled radius in Foundation 5 - which is when this portd over from. We can instead add it to the base style and make it a variable set to global radius

File: src/building-blocks/card-info/card-info.html (hunk 2 above)
Comment (rafibomb) [inline src/building-blocks/card-info/card-info.html]: it is for now ;) until we get more cards

**Pull request merged**